### PR TITLE
fix(console): close modal on tab changed on user details page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

fix(console): close modal on tab changed on user details page

Bug:
bug-user

## Testing

fixed-user

### CODE REVIEW

File: packages/console/src/pages/UserDetails/index.tsx [hunk 1]
```diff
@@ -55,6 +55,11 @@ const UserDetails = () => {
     return userDetailsParser.toLocalForm(data);
   }, [data]);
 
+  useEffect(() => {
+    setIsDeleteFormOpen(false);
+    setIsResetPasswordFormOpen(false);
```
Comment (charIeszhao) [inline packages/console/src/pages/UserDetails/index.tsx:60]: I think you only need to detect if the route has been changed. If changed, close all modals. You don't necessarily have to check if the current route ends with `/logs`

Comment (simeng-li) [reply packages/console/src/pages/UserDetails/index.tsx:60]: Agreed this should be a general fix PR. All the page-level models should close automatically when navigating between pages.  
P.S. the PR title is a little bit misleading as well.

Comment (xiaoyijun) [reply packages/console/src/pages/UserDetails/index.tsx:60]: updated.

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/pages/UserDetails/index.tsx [hunk 2]
```diff
@@ -32,8 +32,8 @@ import * as styles from './index.module.scss';
 import { userDetailsParser } from './utils';
 
 const UserDetails = () => {
-  const location = useLocation();
-  const isLogs = location.pathname.endsWith('/logs');
+  const { pathname } = useLocation();
+  const isLogs = pathname.endsWith('/logs');
```
Comment (charIeszhao) [inline packages/console/src/pages/UserDetails/index.tsx:36]: Is this `isLogs` still used anywhere?

Comment (xiaoyijun) [reply packages/console/src/pages/UserDetails/index.tsx:36]: Yes. The origin code uses it.

**Pull request merged**